console/src/components/Services/Data/TableRelationships/RemoteRelationships/components/EditorCollapsed.tsx

```diff
@@ -10,6 +10,11 @@ const Collapsed: React.FC<Props> = ({ relationship }) => {
   if (!relationship) {
     return null;
   }
+  const parseRelationship = parseRemoteRelationship(relationship);
+  const relationMap = () => {
+    const fields = parseRelationship.remoteFields.map(field => field.name);
+    return ['', ...fields].join(' →');
```

Comment:
We should probably add more spacing here. The deeper the nesting, the more congested it might look. See image below for context.
 
![image](https://user-images.githubusercontent.com/6604943/91582438-701edf80-e96d-11ea-9840-26dcabbd7148.png)